Automattic/wp-calypso
# Added theme search filters to the logged in theme showcase search bar

Related to https://github.com/Automattic/wp-calypso/issues/82640

By removing the search filters from both LoTS and LiTS we inadvertently removed a powerful support tool used by HEs. To ensure sustainable and high-quality support, we will restore them (only on LiTS) while we complete the Search Experience project.

## Proposed Changes

* Added back the filters to the logged-in Theme Showcase

## Testing Instructions



* Open the Calypso Live Link that you can find in this PR
* Navigate to the **LoTS**, and click on the search bar. You **shouldn't see** the dropdown menu with filters.
* If you add filters to the URL the added filters should continue to work.
* Navigate to the **LiTS**, and use the search bar. You **should see** the dropdown menu with filters.
* If you add filters to the URL the added filters should continue to work.

## Pre-merge Checklist
